pkgs/build-support/zig/build-zig-package/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, zig, mktemp }:
```

not sure where mktemp came from

```suggestion
{ lib, stdenv, zig }:
```